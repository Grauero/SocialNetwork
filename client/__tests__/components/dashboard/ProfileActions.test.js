import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { mount } from 'enzyme';
import toJSON from 'enzyme-to-json';

import ProfileActions from '../../../src/components/dashboard/ProfileActions';

const component = mount(
  <BrowserRouter>
    <ProfileActions />
  </BrowserRouter>
);

it('matches snapshot', () => {
  expect(toJSON(component)).toMatchSnapshot();
});
