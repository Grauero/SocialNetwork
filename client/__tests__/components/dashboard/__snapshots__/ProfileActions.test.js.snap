// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<BrowserRouter>
  <Router
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "createHref": [Function],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "length": 1,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <ProfileActions>
      <div
        className="btn-group mb-4"
        role="group"
      >
        <Link
          className="btn btn-light"
          replace={false}
          to="/edit-profile"
        >
          <a
            className="btn btn-light"
            href="/edit-profile"
            onClick={[Function]}
          >
            <i
              className="fas fa-user-circle text-info mr-1"
            />
            Edit Profile
          </a>
        </Link>
        <Link
          className="btn btn-light"
          replace={false}
          to="/add-experience"
        >
          <a
            className="btn btn-light"
            href="/add-experience"
            onClick={[Function]}
          >
            <i
              className="fab fa-black-tie text-info mr-1"
            />
            Add Experience
          </a>
        </Link>
        <Link
          className="btn btn-light"
          replace={false}
          to="/add-education"
        >
          <a
            className="btn btn-light"
            href="/add-education"
            onClick={[Function]}
          >
            <i
              className="fas fa-graduation-cap text-info mr-1"
            />
            Add Education
          </a>
        </Link>
      </div>
    </ProfileActions>
  </Router>
</BrowserRouter>
`;
